docs/docs/mjolnir/traffic_tiles.md

```diff
@@ -0,0 +1,73 @@
+# Traffic in Valhalla
+
+Traffic data for routing engines can be divided into two different types:
+
+- *Historical traffic* describes the typically observed speed on a road at a specific point in time. So e.g. the typical speed on a Monday morning at 8 o'clock.
+- *Live traffic* describes the currently observed speed on a road according to the actual traffic situation.
+
+Valhalla supports both types of traffic for all APIs, but there are some excluded use-cases. First, traffic is not supported for multimodal costing. Furthermore, traffic is currently not supported for `arrive_by` timestamps in the `source_to_targets` API. For more information about how to route with traffic data and time information, check the respective API documentation. In the following it will be explained how to integrate historical and live traffic data into the valhalla graph.
```

Comment:
> traffic is currently not supported for `arrive_by` timestamps in the `source_to_targets` API

That's only half true: using timedistance matrix (and less targets than sources), we do support `arrive_by`, so it's mostly a configuration item.